pkg/cloudprovider/cloudprovider.go

```diff
@@ -358,9 +358,7 @@ func (c *CloudProvider) instanceToNodeClaim(i *instance.Instance, instanceType *
 	labels[karpv1.CapacityTypeLabelKey] = i.CapacityType
 	if v, ok := i.Tags[karpv1.NodePoolLabelKey]; ok {
 		labels[karpv1.NodePoolLabelKey] = v
-	}
-	if v, ok := i.Tags[karpv1.ManagedByAnnotationKey]; ok {
-		annotations[karpv1.ManagedByAnnotationKey] = v
+		annotations[karpv1.NodePoolLabelKey] = v
```

Comment:
Why do we need the NodePool in the annotations?

Reply:
Annotation was to one for one swap with the `managed-by` one and to satisfy the filter in `garbagecollection/controller.go` but if it is no longer used then we can drop it.